src/CMakeLists.txt

```diff
@@ -35,7 +33,6 @@ include_directories(
 message(STATUS "Compiling Radium with 3rdparty include dirs : ${EIGEN3_INCLUDE_DIR} -- ${GLBINDING_INCLUDE_DIR} -- ${GLM_INCLUDE_DIR} -- ${GLOBJECTS_INCLUDE_DIR}-- ${OPENMESH_INCLUDE_DIR}" )
 
 if( RADIUM_ASSIMP_SUPPORT )
-        set( RADIUM_IO_IS_INTERFACE FALSE )
     include_directories( ${ASSIMP_INCLUDE_DIR} )
```

Comment:
This is mandatory . If we use assimp, the IO is not an interface. Please put it back

Reply:
As I said, I (wrongly) didn't put the camera loader as an option, thus IO was never an interface